makelove/config.py

```diff
@@ -87,6 +87,13 @@
             "artifacts": val.ValueOrList(val.Choice("appdir", "appimage")),
         }
     ),
+    "macos": val.Section(
+        {
+            "love_binaries": val.Path(),
+            "icon_file": val.Path(),
+            "app_metadata": val.Dict(val.String(), val.String()),
+        }
+    ),
```

Comment:
I'm sorry, but I forgot the specifics. We need a separate `icon_file` here, because we cannot convert with PIL (on non-Mac systems), correct? Could you please add a comment about that (that we can't convert and need a separate icns) to `makelove_full.toml`?

Reply:
the mac icon is optional, but mac icons generally look different so this is a way to specify a mac-specific icon.

for example Mac icons generally have a drop shadow and 5% of the pixels around the edges are not used for the icon object (just the shadow, etc). It looks pretty bad if you have a square icon that goes edge to edge on mac.